src/components/dropzone/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import View from '../view';
+
+class DropZone extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      active: false,
+    };
+    this.ref = React.createRef();
+  }
+  componentDidMount() {
+    this.ref.current.addEventListener('mouseup', this.onDragLeave);
+    this.ref.current.addEventListener('dragenter', this.onDragEnter);
+    this.ref.current.addEventListener('dragover', this.onDragOver);
+    this.ref.current.addEventListener('drop', this.onDrop);
+  }
+  componentWillUnmount() {
+    this.ref.current.removeEventListener('mouseup', this.onDragLeave);
+    this.ref.current.removeEventListener('dragenter', this.onDragEnter);
+    this.ref.current.addEventListener('dragover', this.onDragOver);
+    this.ref.current.removeEventListener('drop', this.onDrop);
+  }
+
+  // TODO: Extract data from files.
+  onDrop = e => {
+    e.preventDefault();
+    const files = e.dataTransfer.files;
+    window.alert(`Files dropped: ${files}`);
```

Comment:
This is just printing `Files dropped: [object FileList]` which isn't meaningful

Reply:
@michael1011 Yeah it was just so you could see what it does. I will replace it with real functionality tomorrow.